tutorials/llama_index_multimodal_image_reasoning.ipynb

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
<p>I would combine this with the below, nix the TBDs and not use the warning symbol for llama-index</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Oh yes they were meant to be combined my bad</p>